libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -537,6 +537,20 @@ namespace SohImGui {
         }
     }
 
+    void EnhancementCombo(const std::string& name, const char* cvarName, std::vector<std::string> items, int defaultValue) {
```

Comment:
```suggestion
    void EnhancementCombo(const std::string& name, const char* cvarName, const std::vector<std::string>& items, int defaultValue) {
```
This should be a const reference to prevent copies.

Reply:
Good point, changed that with the next commit.